edge/pkg/edgestream/tunnel.go

```diff
@@ -172,7 +191,7 @@ func (s *TunnelSession) Serve() error {
 			return fmt.Errorf("close tunnel stream connection, error:%s", string(mess.Data))
 		}
 
-		if mess.MessageType < stream.MessageTypeData {
+		if (mess.MessageType < stream.MessageTypeData) || (mess.MessageType == stream.MessageTypeAttachConnect) {
```

Comment:
`mess.MessageType >= stream.MessageTypeAttachConnect` maybe better for expansion

Reply:
> ```
> const (
> 	MessageTypeLogsConnect MessageType = iota
> 	MessageTypeExecConnect
> 	MessageTypeMetricConnect
> 	MessageTypeData
> 	MessageTypeRemoveConnect
> 	MessageTypeCloseConnect
> 	MessageTypeAttachConnect
> ```
> 
> What should this field be adjusted to be greater than or equal to if it is increased by 1 each time here?

I guess that in order to ensure compatibility, new message type will be added at the end. If we add more capabilities of kubectl in the future, we don't need to change that judgment, just using `mess.MessageType >= stream.MessageTypeAttachConnect` is OK.  It's just my suggestion.

Reply:
I get it .thanks